src/card.css

```
@@ -0,0 +1,101 @@
+:root {
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --button-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
```

Create variables for all colors)
```suggestion
  background-color: var(--color-white);
```